README.md

```diff
@@ -1,2 +1,32 @@
-# codecept-js-playwright-browserstack
-Sample code for running sessions with Codecept Playwright on Browserstack
+# CodeceptJS-Playwright-Browserstack
+
+# Integrating Playwright <a href="https://playwright.dev/docs/intro"><img src="https://playwright.dev/img/playwright-logo.svg" alt="Node.js" height="22" /></a>  and CodeceptJS <a href="https://codecept.io/"><img src="https://codecept.io/logo.svg" alt="Node.js" height="22" /></a> with BrowserStack <a href="https://browserstack.com"><img src="https://raw.githubusercontent.com/browserstack/MSTest-playwright-browserstack/main/assets/browserstack.png" alt="BrowserStack" height="22" /></a>
+
+
+## Setup
+*  Clone the repo `git clone -b sdk https://github.com/browserstack/codecept-js-playwright-browserstack.git`.
```

Comment:
Let's confirm the branch `main` or `sdk`. Rest lgtm

Reply:
It should be main ( got this info from PM) , changed the readme accordingly